### DISCUSSION TITLE
Understand the purpose of DLL's of provided in the release

### DISCUSSION THREAD

**Discussion by dr-dba** [orbitalquark/scintillua#42] [Q&A] [answered]
There is a pair of DLL's in the provided Scintillua release:
LexLPeg.dll
LexLPeg32.dll

Is it precompiled Scintillua helpers (scintilla engine?),
either these are Scintillua itself compiled for Windows?

**Comment by orbitalquark** [comment, marked as answer]
As mentioned in the README: "Scintillua adds dynamic Lua LPeg lexers to Scintilla. It is the quickest way to add new or customized syntax highlighting and code folding for programming languages to any Scintilla-based text editor or IDE. Scintillua was designed to be dropped into or compiled with any Scintilla environment."

Those DLLs are external lexers for use in a Scintilla-based editor on Windows. They are lexers only, completely separate from Scintilla, and do not provide any sort of Lua interface to Scintilla.
